2/Dockerfile.rhel7

```diff
@@ -54,6 +54,10 @@ RUN rm /opt/openshift/base-plugins.txt && \
     # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
     # HPI file created by rpm to /opt/openshift/plugins folder.
     for FILENAME in /usr/lib/jenkins/*hpi ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
+    # for backward compatability with pre-3.6 installs leveraging a PV, where rpm installs went to /usr/lib64/jenkins, we are
+    # establishing a symbolic link for that guy as well, so that existing plugins in JENKINS_HOME/plugins pointing to /usr/lib64/jenkins
+    # will subsequently get redirected to /usr/lib/jenkins; it is confirmed that the 3.7 jenkins RHEL images do *NOT* have a /usr/lib64/jenkins path
+    ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
```

Comment:
you should create this symlink *before* installing the rpms, so that any rpm that installs to /usr/lib64/jenkins lands them in the symlinked dir.